### ISSUE TITLE
Accessibility - WCAG 1.1.1 Non-text Content

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#141]
There should be a meaningful and equivalent alternative for all non-text content, such as images, graphics, objects, graphic controls in forms and hotspots in image maps.

Carousel should have alt-dummy too.
![image](https://cloud.githubusercontent.com/assets/7510842/3226751/9f66c0dc-f072-11e3-9d6e-e0d7ae66280f.png)

Original version was ok
